lib/matplotlib/widgets.py

```diff
@@ -3267,97 +3338,120 @@ def corners(self):
         Corners of rectangle in data coordinates from lower left,
```

Comment:
Is this docstring still correct?
When I rotate a square selector by 90 degree, I would get the same values, but it is not the case:
before:
```python
(array([0.34274322, 0.59980032, 0.59980032, 0.34274322]),
 array([0.39714243, 0.39714243, 0.74211084, 0.74211084]))
```
after
```python
(array([0.60062956, 0.5989657 , 0.34191398, 0.34357784]),
 array([0.39826248, 0.74322366, 0.74099079, 0.3960296 ]))
```